Common/Packets/BacktestNodePacket.cs

```diff
@@ -112,6 +117,7 @@ public BacktestNodePacket(int userId, int projectId, string sessionId, byte[] al
             ProjectId = projectId;
             UserPlan = userPlan;
             Name = name;
+            CashAmount = startingCapital;
```

Comment:
startingCapital parameter was already passed in constructor, but was not used. Now assigned to new property created.

Reply:
We might keep this constructor definition which defaults to `USD` currency and calls a new one which receives a `List<CashAmount>`.